docs/pages/desktop-access/reference/clipboard.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Clipboard Sharing
+description: Using Clipboard Sharing with Teleport Desktop Access.
+---
+
+Teleport Desktop Access supports copying and pasting text between your browser
+and a remote Windows Desktop.
+
+<Adminition type="note">
+This feature requires a Chromium-based browser such as Google Chrome, Brave,
+or Microsoft Edge.
+</Admonition>
+
+Clipboard sharing is configured via Teleport's RBAC system. It is enabled by default,
+but can be disabled by setting the following role option on one or more of a user's
+roles:
+
+```yaml
+desktop_clipboard: false
+```
```

Comment:
```suggestion
```yaml
kind: role
version: v4
metadata:
  description: Clipboard disabled
  name: clipboard_disabled
spec:
  options:
    desktop_clipboard: false
```
```

Reply:
That's a valid role spec too, the user can just `tctl create -f` that role and assign it to any user they want to block clipboard sharing from.